src/const.ts

```diff
@@ -135,13 +135,13 @@ export const ETH_NODE_URL = ethNodeUrl
 
 export const STORAGE_KEY_LAST_PROVIDER = 'lastProvider'
 
-export const STORAGE_KEY_DISABLED_TOKENS_ADDRESSES = 'disabledTokens'
-
-export const GP_ORDER_TX_HASHES = {
-  1: 'GP_ORDER_TX_HASHES_1',
-  4: 'GP_ORDER_TX_HASHES_4',
+export const STORAGE_PENDING_ORDER_TX_HASHES = {
```

Comment:
not sure if this belong here, cause doesn't look like a general const that different components may use. It looks just like part of a reducers implementation